lib/rules/function-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: true,
+	skipBasicChecks: true,
+
+	accept: [
+		{
+			code: 'a { transform: translate(1px); }',
+		},
+		{
+			code: 'a { transform: --custom-function(1px); }',
+		},
+		{
+			code: 'a { transform: scale(0.5) translate(-100%, -100%); }',
+		},
+		{
+			code: 'a { $list: (list) }',
+		},
```

Comment:
Let's move this SCSS into its own `testRule` that uses the `postcss-scss` syntax as we strive to keep standard and non-standard syntax separate.